lib/hooks/endorsed/project.js

```diff
@@ -0,0 +1,54 @@
+const { get, merge } = require('lodash');
+const { BadRequestError } = require('@asl/service/errors');
+const { awaitingEndorsement, withInspectorate } = require('../../flow/status');
+const Messager = require('../../messager');
+
+module.exports = settings => model => {
+  const messager = settings.StubMessager ? settings.StubMessager : Messager(settings);
```

Comment:
It feels like this is something that should be done inside the constructor given we pass the settings in, and not leaked out here.

Reply:
Also creating the messager instance should probably occur outside the inner function.